providers/github/connection/platform.go

```diff
@@ -19,26 +19,29 @@ const (
 )
 
 var (
-	GithubRepoPlatform = &inventory.Platform{
-		Name:    "github-repo",
-		Title:   "GitHub Repository",
-		Family:  []string{"github"},
-		Kind:    "api",
-		Runtime: "github",
-	}
-	GithubUserPlatform = &inventory.Platform{
-		Name:    "github-user",
-		Title:   "GitHub User",
-		Family:  []string{"github"},
-		Kind:    "api",
-		Runtime: "github",
-	}
-	GithubOrgPlatform = &inventory.Platform{
-		Name:    "github-org",
-		Title:   "GitHub Organization",
-		Family:  []string{"github"},
-		Kind:    "api",
-		Runtime: "github",
+	GithubRepoPlatform = inventory.Platform{
+		Name:                  "github-repo",
+		Title:                 "GitHub Repository",
+		Family:                []string{"github"},
+		Kind:                  "api",
+		Runtime:               "github",
+		TechnologyUrlSegments: []string{"github", "repo"},
```

Comment:
I'm not sure about this:
Shouldn't "repo" match something in the UrlTree? There we have "repository".

Reply:
It is, "*" means that all values that do not have "org" are repositories